README.md

```diff
@@ -510,6 +510,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Retrofit](http://square.github.io/retrofit/) - Type-safe REST client.
 * [Spark](http://sparkjava.com/) - Sinatra inspired framework.
 * [Swagger](http://swagger.io/) - Swagger is a specification and complete framework implementation for describing, producing, consuming, and visualizing RESTful web services.
+* [Blade](https://github.com/biezhi/blade) - Blade is a simple, elegant Java Web Framework!
```

Comment:
Please remove it from this section since it explicitly states that it wants to be a MVC webframework.